src/gui/static/src/app/components/layout/hardware-wallet/hw-passphrase-help-dialog/hw-passphrase-help-dialog.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+import { MatDialogRef } from '@angular/material/dialog';
+
+@Component({
+  selector: 'app-hw-passphrase-help-dialog',
+  templateUrl: './hw-passphrase-help-dialog.component.html',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163608)
```suggestion
  templateUrl: "./hw-passphrase-help-dialog.component.html",
```